### ISSUE TITLE
TabError: inconsistent use of tabs and spaces in indentation in 1.2.1a

### ISSUE THREAD

**Issue by oopiicaa** [liaan/broadlink_ac_mqtt#96]
With the latest version, 1.2.1a, this comes up after starting monitor.py:
```
Traceback (most recent call last):
  File "monitor.py", line 9, in <module>
    import broadlink_ac_mqtt.AcToMqtt as AcToMqtt
  File "/home/ac2mqtt/ac2mqtt-tar-extracted/broadlink_ac_mqtt-1.2.1a/broadlink_ac_mqtt/AcToMqtt.py", line 14, in <module>
    import broadlink_ac_mqtt.classes.broadlink.ac_db as broadlink
  File "/home/ac2mqtt/ac2mqtt-tar-extracted/broadlink_ac_mqtt-1.2.1a/broadlink_ac_mqtt/classes/broadlink/ac_db.py", line 504
    def set_display(self,value):
                               ^
TabError: inconsistent use of tabs and spaces in indentation
```

**Comment by liaan** [comment]
offs.. now I know why I stopped coding python

let me check.. sorry, did not test the Pull requests

**Comment by liaan** [comment]
ok, hopefully fixed now, can you please check 1.2.1b

**Comment by oopiicaa** [comment]
Thank you very much, it's working now.
